Rakefile

```diff
@@ -10,4 +10,33 @@ RuboCop::RakeTask.new do |task|
   task.requires << 'rubocop-rspec'
 end
 
+desc 'Build the manual'
+file 'man/colorls.1' do
+  require 'colorls'
+  require 'ronn'
+
+  flags = ColorLS::Flags.new
+  attributes = {
+    date: Time.now,
+    manual: 'colorls Manual',
+    organization: "colorls #{ColorLS::VERSION}"
+  }
+  doc = Ronn::Document.new(nil, attributes) do
+    template = IO.read('man/colorls.1.ronn')
+
+    section = ''
+    flags.options.each do |o|
+      section += <<OPTION
+  * `#{o.flags.join('`, `')}`:
+     #{o.desc.join("<br>\n")}
+
+OPTION
+    end
+    template.sub('{{ OPTIONS }}', section)
+  end
+  IO.write('man/colorls.1', doc.convert('roff'))
+end
+
 task default: %w[spec rubocop]
+
+task install: 'man/colorls.1'
```

Comment:
Works well. The formed man file is automatically exposed to manpath - cool! Confirmed for Mac.

One question though - if there's already a `man/colorls.1` file, `rake install`ing doesn't re-form the man file. So, any changes in the ronn file doesn't get reflected unless the existing `man/colorls.1` is manually removed? Can a `File.delete('man/colorls.1')` be included within the task?

Reply:
> Can the man/colorls.1 be gitignored?

Yes, good point!

I'll look into the rake task issue, probably need to define a phony target...